File: city-weather-angular/src/app/model/CityWeatherReport.ts
export class CityWeatherReport{
id ? : string;
city ?: string;
condition?: string;
localTime? : string;
maxTemperature? : string;
minTemperature? : string;
imageUrl? : string;

constructor(id?:string,city?:string, localTime?:string, condition?:string, maxTemperature?:string,minTemperature?:string,imageUrl?:string){
this.id=id;
this.city = city;
this.localTime = localTime;
this.condition = condition;
this.maxTemperature = maxTemperature;
this.minTemperature = minTemperature;
this.imageUrl = imageUrl;
}

}

File: city-service/src/main/java/com/mindfire/cityservice/dto/CityWeatherReport.java
package com.mindfire.cityservice.dto;

import lombok.Data;

@Data
public class CityWeatherReport {
	private String id;
	private String city;
	private String localTime;
	private String condition;
	private String maxTemperature;
	private String minTemperature;
	private String imageUrl;

	public CityWeatherReport(String id,String city, String localTime, String condition, double maxTemperature, double minTemperature, String imageUrl) {
		
		super();
		this.id=id;
		this.city = city;
		this.localTime= localTime;
		this.condition = condition;
		this.maxTemperature = Double.toString(maxTemperature);
		this.minTemperature = Double.toString(minTemperature);
		this.imageUrl = imageUrl;
	}

	
	
}
